src/tools/adapter/Adapter.ts

```diff
@@ -0,0 +1,8 @@
+import { UbiquitousLanguageJson } from '../../model/UbiquitousLanguageJson';
+import { UbiquitousLanguageDto } from '../../model/DTO/UbiquitousLanguageDto';
+
+export interface IAdapter {
```

Comment:
I don't think this Adapter is relevant for now

Reply:
YAGNI or hexagonal architecture ? That's the dilemma